executors/kubeexec/kubeexec.go

```diff
@@ -114,7 +114,7 @@ func NewKubeExecutor(config *KubeConfig) (*KubeExecutor, error) {
 	}
 
 	if config.MountOpts == "" {
-		k.MountOpts = MNTOPTS//"rw,inode64,noatime,nouuid"
+		k.MountOpts = cmdexec.MNTOPTS
```

Comment:
Pardon my earlier comments, but this indicates the commits should be squashed.

Reply:
@phlogistonjohn made the requested changes.